README.md

```diff
@@ -4,11 +4,23 @@ This R package implements the methods proposed in Imai, K. and Khanna, K. (2016)
 
 ### Using wru
```

Comment:
TODO: Readme needs more details about changes to the package and guidance on what to use, and a bit on why. Possibly a vignette would be appropriate.